knowledge_storm/decorators/cache_decorator.py

```
@@ -0,0 +1,217 @@
+"""
+Cache decorator for research planning operations.
+
+Provides a clean separation of caching concerns from core business logic
+following the Single Responsibility Principle.
+"""
+
+import asyncio
+import functools
+import hashlib
+import logging
+from typing import Any, Callable, Dict, Optional, TypeVar, Union
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)